packages/components/src/components/ExternalCard/elements.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+export const StyledWrapper = styled(Stack)<{ thumbnail: string }>`
+  position: relative;
+  overflow: hidden;
+  height: 156px;
+  width: 276px;
+  background: #999999;
+  border-radius: 4px;
+
+  &:after {
+    position: absolute;
+    content: '';
+    display: block;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-image: url(${props => props.thumbnail});
+    background-size: cover;
+    background-position: center;
+    transform: scale(1);
+    transition: transform ease-in-out;
+    transition-duration: 100ms;
+  }
+
+  &:hover::after {
+    transform: scale(1.01);
+  }
```

Comment:
Missing `focus` or `focus-visible` styles and `hover` "overlay"-color change.

Reply:
<img width="695" alt="Screen Shot 2022-12-08 at 19 17 44" src="https://user-images.githubusercontent.com/24959348/206579048-9e25a697-8f70-461a-9410-bf3f42d1e4a2.png">